lib/active_encode/engine_adapters/s3.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
```

Comment:
This module looks like a duplicate of methods in the adapter class.  Were you intending to make a shared module for both the MediaConvert adapter and the ElasticTranscode adapter?  Or can this module be thrown away?

Reply:
Ah yes. This was an initial attempt at abstraction that I intended to throw away.